tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -3338,11 +3338,14 @@ class HLSLExternalSource : public ExternalSemaSource {
 
   // Adds intrinsic function declarations to the "vk" namespace.
   // It does so only if SPIR-V code generation is being done.
+  // Assumes the implicit "vk" namespace has already been created.
   void AddVkIntrinsicFunctions() {
     // If not doing SPIR-V CodeGen, return.
     if (!m_sema->getLangOpts().SPIRV)
       return;
 
+    DXASSERT(m_vkNSDecl);
```

Comment:
you'll either need to add a second argument with the output message like DXASSERT(m_vkNSDecl, "VK namespace hasn't been created") or replace it with DXASSERT_NOMSG(). I prefer the former.

Reply:
Done